src/modules/AuthPayload/model.ts

```diff
@@ -0,0 +1,9 @@
+import { objectType } from 'nexus/dist';
```

Comment:
I suggest to put all of those auth-related files into a single `module` called `Auth` or `User`